static/lib-utils.ts

```diff
@@ -0,0 +1,55 @@
+import _ from 'underscore';
```

Comment:
Should we maybe merge this with `static/utils.ts`?

Reply:
Maybe.. In general I don't like calling things utils at all. I'd rather give it a good name. lib-utils is slightly better than just plain utils